services/convert/convert.go

```diff
@@ -308,40 +308,43 @@ func ToTeam(ctx context.Context, team *organization.Team, loadOrg ...bool) (*api
 
 // ToTeams convert models.Team list to api.Team list
 func ToTeams(ctx context.Context, teams []*organization.Team, loadOrgs bool) ([]*api.Team, error) {
-	if len(teams) == 0 || teams[0] == nil {
-		return nil, nil
-	}
-
 	cache := make(map[int64]*api.Organization)
-	apiTeams := make([]*api.Team, len(teams))
-	for i := range teams {
-		if err := teams[i].LoadUnits(ctx); err != nil {
+	apiTeams := make([]*api.Team, 0, len(teams))
+	for _, t := range teams {
+		if t == nil {
+			// Could be nil if teams was created with make([]*Team, x)
+			continue
+		}
```

Comment:
I think that was the reason for the `teams[0] == nil` check. Can't tell if that check is still necessary.

Reply:
I did not find a place which creates arrays with size and uses `append` afterwards. Should I remove this check then? Usually we don't have such checks.

Reply:
Especially as this check silently masks a failure somewhere else.
I'm rather against including it.
If we do make a mistake somewhere, it should panic, so that we can then fix the problem.